pycode/test/test_epidata_get_divi_data.py

```diff
@@ -45,41 +45,43 @@ class TestGetDiviData(fake_filesystem_unittest.TestCase):
 
     # strings for read, download and update data
     test_string1 = ("""{"bundesland":1,"gemeindeschluessel":1001,"anzahl_meldebereiche":2,"faelle_covid_aktuell":0,\
-"faelle_covid_aktuell_beatmet":0,"anzahl_standorte":2,"betten_frei":48,"betten_belegt":34,\
+"faelle_covid_aktuell_invasiv_beatmet":0,"anzahl_standorte":2,"betten_frei":48,"betten_belegt":34,\
```

Comment:
Why did you change the column names? The program should rename it since date is before 31.03.2021.